### DISCUSSION TITLE
copilot sign in not working on Pycharm

### DISCUSSION THREAD

**Discussion by nagaraj1497** [community/community#114453] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I installed GitHub copilot plugin in my pycharm editor. Later I tried signing in, I loggedin through my credentials, and authorised co-pilot on safari browser. But sign in is not working. I'm attaching the screenshot for reference. This is showing from past 1 hour.
<img width="472" alt="Screenshot 2024-03-27 at 12 44 10 PM" src="https://github.com/community/community/assets/64498650/31a73b2c-2859-4e43-bbb3-c671b5a9c4ab">

**Comment by Akash1134** [comment]
Hi @nagaraj1497 ,

Possibly, there is a configuration change to be made in your JetBrains IDE. We sometimes see issues with proxy configurations - For more information, read [Configuring network settings for GitHub Copilot](https://docs.github.com/enterprise-cloud@latest/copilot/configuring-github-copilot/configuring-network-settings-for-github-copilot#title-h1) in the GitHub Docs.

If you or your organization employs security measures like a firewall or proxy server, it may be beneficial to include [certain domain URLs](https://docs.github.com/en/enterprise-cloud@latest/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot) in an "allowlist" and open specific ports and protocols. Doing so will enhance your installation and usage of GitHub Copilot for an optimal experience. Also, please refer this [guide](https://docs.github.com/en/enterprise-cloud@latest/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot).

Thanks!
